bin/install-if-deps-outdated.js

```diff
@@ -24,7 +24,7 @@ const needsInstall = () => {
 };
 
 if ( needsInstall() ) {
```

Comment:
It seems like `needsInstall` will need some reworking. It would need to take into account the root shrinkwrap as well as locks in any packages, right?

Reply:
reworked